test/functional/tests/initialize/test_negative_load.py

```diff
@@ -59,3 +59,49 @@ def test_load_occupied_id():
         cores = caches[0].get_core_devices()
         if len(cores) != 0:
             TestRun.LOGGER.error("Inappropriate number of cores after load!")
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_add_cached_core():
+    """
+        title: Negative test for adding already used core to a cache.
+        description: |
+          Negative test of the ability to add the core to the cache twice to the same cache
```

Comment:
I don't think `add the core **the cache twice to the same cache**` was your intention :)